sdk/storage/azure-storage-blob/src/test/java/com/azure/android/storage/blob/StorageBlobClientTest.java

```diff
@@ -103,27 +103,35 @@ public void getBlobsInPage_withCallback() {
 
         mockWebServer.enqueue(mockResponse);
 
-        ServiceCall serviceCall = storageBlobClient.getBlobsInPage(null,
+        CountDownLatch latch = new CountDownLatch(1);
```

Comment:
Tests were not failing, but there is a possibility of "synchronous" test methods exit before the "async API call" it makes completes, this is because, the test method and callbacks execute in different threads, for 
correctness, we need some way to synchronize these two threads; the count-latch is used to make the sync test method wait until async API completes.

Reply:
Apps (customers) don’t have to use this unless they mix its sync logic with our async API, but in that case, we’ve sync client.